modules/random.js

```diff
@@ -5,12 +5,11 @@ module.exports = {
       command: function (bot, msg) {
         if (msg.args.length > 2) {
           msg.args.shift()
-          const randint = msg.args[Math.floor(Math.random() * msg.args.length)]
-          return randint.toString()
+          return msg.args[Math.floor(Math.random() * msg.args.length)]
         } else if (msg.args.length === 2) {
           var res = Math.floor(Math.random() * Math.floor(msg.args[1]))
           if (!isNaN(res)) {
-            return res
+            return res.toString
```

Comment:
This will return the actual `Number.prototype.toString` function itself, instead of the output from it. Use.`.toString()` to actually call the function.